axis.go

```diff
@@ -5,14 +5,17 @@
 package plot
 
 import (
-	"fmt"
 	"image/color"
 	"math"
+	"strconv"
 
 	"github.com/gonum/plot/vg"
 	"github.com/gonum/plot/vg/draw"
 )
 
+// sane is a sane level of float precision for a plot.
+const sane = 4
```

Comment:
why not just call it `fprec` ? or `fltprec` ? `floatPrec`? 
I'd say `sane` is a bit insane :}